src/theme/components/template.ts

```diff
@@ -14,79 +14,74 @@ const Template = defineStyleConfig({
       minWidth: "288px",
       m: "0 auto",
       p: responsiveMargin,
-      gridTemplateAreas: `
-        "breakout"
-        "top"
-        "main"
-        "bottom"
-      `,
-      gridTemplateColumns: "100%",
+      gridTemplateColumns: "repeat(12, 1fr)",
       gridTemplateRows: "auto",
       columnGap: responsiveGap,
       "& > *:not(:last-child)": { mb: responsiveGap },
+
+      /** Set the full and main content areas to span the full width of the
+       * content area from a mobile-first viewpoint. Using -1 for the "last
+       * column" value ensures that these elements span all columns in the grid.
+       * */
+      "& .reservoir-template-full": {
+        gridColumn: "1 / -1",
+      },
+      "& .reservoir-template-main": {
+        gridColumn: "1 / -1",
+      },
```

Comment:
If you change this to:

```
 "& .reservoir-template-main": {
        gridColumn: { base: "1 / -1" },
      },
```

then you don't have to include `base: "1 / -1"` in the variants below. For example, sidebarLeft would just have:

```
   "& .reservoir-template-main": {
        gridColumn: {
          sm: "7 / -1",
          md: "5 / -1",
          lg: "4 / -1",
        },
      },
```